draft-ietf-taps-transport-security.md

```diff
@@ -173,9 +176,12 @@ even where their interfaces to transport and applications are equivalent. Even b
 protocols have subtly different guarantees and vulnerabilities. Thus, any implementation needs to only
 use the set of protocols and algorithms that are requested by applications or by a system policy.
 
+Different security protocols also can use incompatible notions of peer identity and authentication, and
```

Comment:
I would change the order and write "Different security protocols can also use", but I'm not sure either is more correct than the other.

Reply:
Eh, I think it's fine as is.